Mathlib/Analysis/Calculus/FDeriv/Basic.lean

```diff
@@ -336,7 +336,7 @@ theorem hasFDerivAt_iff_isLittleO_nhds_zero :
 
 nonrec theorem HasFDerivAtFilter.mono (h : HasFDerivAtFilter f f' x L₂) (hst : L₁ ≤ L₂) :
     HasFDerivAtFilter f f' x L₁ :=
-  .of_isLittleO <| h.isLittleO.mono hst
+  .of_isLittleOTVS <| h.isLittleOTVS.mono hst
```

Comment:
Should we change typeclass assumptions here, or do you want to do it in another PR (e.g., wait until we can generalize more theorems)?

Reply:
In another PR; I want to generalize as much as possible without having to shuffle things around later.